src/uucore/Cargo.toml

```diff
@@ -25,6 +25,7 @@ dns-lookup = { workspace = true, optional = true }
 dunce = { version = "1.0.4", optional = true }
 wild = "2.2.1"
 glob = { workspace = true }
+lazy_static = "1.4.0"
```

Comment:
I would use the most recent version, `1.5.0`, unless there is some specific reason for using the older version.

Reply:
I've used this version as it's the one currently used as an indirect dependency and I wanted to minimize the impact of the change.

I will update it :)

Reply:
That makes sense, so you can also leave it as it is and let `renovate` do the update later :)

Reply:
Ok, will that then ^^